src/views/List.jsx

```diff
@@ -40,7 +45,13 @@ export function List({ data }) {
 						)
 						.map((item, id) => <ListItem key={id} name={item.name} />)
 				) : (
-					<h1>You have no items in your list!</h1>
+					<>
+						<h1>Welcome to your "{listName}" list. </h1>
```

Comment:
If a user has multiple lists and navigates to one that already has items, the list page doesn't actually tell them which list they're on.  Now that we have the listPath and name available to use on the list page, you might consider it always being displayed, regardless of there being items in the list or not.

Reply:
Great suggestion. Will move the welcome message outside of the conditional the the list name will appear regardless of if there are items in the list. Thanks!